### ISSUE TITLE
Unused variable reported in some indirect objects with 'ToString' methods

### ISSUE THREAD

**Issue by objeck** [objeck/objeck-lang#63]
Unused variable reported in some indirect objects with 'ToString' methods

```
player_armor := @player->GetArmor();
player_armor_name := "{$player_weapon}";```

**Comment by objeck** [comment]
Refer to programs/test/regress/regress0.obs

**Comment by objeck** [comment]
Fixed and released in 87a1efba70caa1eb90e3308c1c12027958aaafc9
